lti_consumer/static/js/xblock_lti_consumer.js

```diff
@@ -92,68 +105,97 @@ function LtiConsumerXBlock(runtime, element) {
             }
         });
 
+        function confirmDialog(message, triggerElement, showCancelButton) {
+            var def = $.Deferred();
+            // Hide the button that triggered the event, i.e. the launch button.
+            triggerElement.hide();
+
+            $('<div id="dialog-container"></div>').insertAfter(triggerElement) // TODO: this will need some cute styling. It looks like trash but it works.
```

Comment:
`TODO: this will need some cute styling. It looks like trash but it works.`

I didn't write this comment, but it's still ugly, so I guess I'll leave it in...